pkgs/development/python-modules/pyside6-fluent-widgets/default.nix

```diff
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchurl,
+  setuptools,
+  pyside6,
+  pysidesix-frameless-window,
+  darkdetect,
+}:
+
+buildPythonPackage rec {
+  pname = "pyside6-fluent-widgets";
+  version = "1.8.1";
+  pyproject = true;
+
+  src = fetchurl {
+    # fetchPypi doesn't able to download files whose file names and package names are inconsistent
+    url = "https://files.pythonhosted.org/packages/f8/69/3be443e8a94d884250658c00441ae4239bbbd2dcff8b1b4a32b82463cbb6/pyside6_fluent_widgets-${version}.tar.gz";
+    hash = "sha256-PMpbGUFN7bb9jKBF+596x4WLqoHjjNN2gX0c4G+lCHo=";
+  };
```

Comment:
Why not `fetchFromGitHub` then?

Reply:
Ah ok.

Reply:
`fetchPypi { pname = "pyside6_fluent_widgets"; version = "1.8.1"; hash = "sha256-PMpbGUFN7bb9jKBF+596x4WLqoHjjNN2gX0c4G+lCHo="; }` seems to work for me?